v19.2/start-a-local-cluster-in-docker-windows.md

```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
+   - `{{page.release_info.docker_image}}:{{page.release_info.version}} start --insecure --join`: The CockroachDB command to [start a node](cockroach-start.html) in the container in insecure mode. The `--join` flag specifies the `hostname` of each node that will initially comprise your cluster. Otherwise, all [`cockroach start`](cockroach-start.html) defaults are accepted. Note that since each node is in a unique container, using identical default ports won’t cause conflicts.
 
 3. Start two more nodes:
 
-    {{site.data.alerts.callout_info}}Again, be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
-
-    ~~~ powershell
-    PS C:\Users\username> docker run -d `
-    --name=roach2 `
-    --hostname=roach2 `
-    --net=roachnet `
-    -p 26257:26257 -p 8080:8080  `
-    -v "//c/Users/<username>/cockroach-data/roach2:/cockroach/cockroach-data"  `
-    {{page.release_info.docker_image}}:{{page.release_info.version}} start `
-    --insecure `
-    --join=roach1,roach2,roach3
-    ~~~
-
-    ~~~ powershell
-    PS C:\Users\username> docker run -d `
-    --name=roach3 `
-    --hostname=roach3 `
-    --net=roachnet `
-    -p 26257:26257 -p 8080:8080  `
-    -v "//c/Users/<username>/cockroach-data/roach3:/cockroach/cockroach-data"  `
-    {{page.release_info.docker_image}}:{{page.release_info.version}} start `
-    --insecure `
-    --join=roach1,roach2,roach3
-    ~~~
+   {{site.data.alerts.callout_info}}Again, be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
+
+   ```powershell
+   PS C:\Users\username> docker run -d `
+   --name=roach2 `
+   --hostname=roach2 `
+   --net=roachnet `
+   -p 26257:26257 -p 8080:8080  `
+   -v "//c/Users/<username>/cockroach-data/roach2:/cockroach/cockroach-data"  `
+   {{page.release_info.docker_image}}:{{page.release_info.version}} start `
+   --insecure `
+   --join=roach1,roach2,roach3
+   ```
+
+   ```powershell
+   PS C:\Users\username> docker run -d `
+   --name=roach3 `
+   --hostname=roach3 `
+   --net=roachnet `
+   -p 26257:26257 -p 8080:8080  `
+   -v "//c/Users/<username>/cockroach-data/roach3:/cockroach/cockroach-data"  `
+   {{page.release_info.docker_image}}:{{page.release_info.version}} start `
+   --insecure `
+   --join=roach1,roach2,roach3
+   ```
 
 4. Perform a one-time initialization of the cluster:
 
-    ~~~ powershell
-    PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
-    ~~~
+   ```powershell
+   PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
+   ```
 
-    You'll see the following message:
```

Comment:
This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.